aggregators/aggregator.go

```diff
@@ -110,7 +110,7 @@ func New(opts ...Option) (*Aggregator, error) {
 		db:             pb,
 		writeOptions:   writeOptions,
 		cfg:            cfg,
-		processingTime: time.Now().Truncate(cfg.AggregationIntervals[0]),
+		processingTime: cfg.OriginProcessTime.Truncate(cfg.AggregationIntervals[0]),
```

Comment:
This would not work. Even if we set the processing time in past, we would just be offsetting the time. The aggregation service takes full control of the processing time and uses it to stamp events and define time ranges for calculating aggregates and overflows. Offsetting the processing time with a static value would just mean that all events are stamped with the offsetted time. This should continue losing data on restarts.